# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
#define a feedback message
string current_state
bool solution_required
uint32 exceptional_case_id

